node/src/actors/chain_manager/mod.rs

```diff
@@ -169,6 +167,10 @@ pub struct ChainManager {
     tx_pending_timeout: u64,
     /// Magic number from ConsensusConstants
     magic: u16,
+    /// External mint address
+    external_address: Option<PublicKeyHash>,
+    /// Mint Percentage to share with the external address
+    external_percentage: u8,
```

Comment:
Are we locally ensuring that `external_percentage > 0` and `external_percentage <= 100`?

That's totally something that is worth being validated at start time, otherwise any blocks produced would be deemed invalid by other peers.

Reply:
if it is 0, nothing happens, but yes, it is needed to limit the external_percentage to 100